docs/users.rst

```diff
@@ -148,7 +148,8 @@ packages
   of the Koji build target used. The following command is useful in determining
   which packages are available in a given Koji build tag:
   ``koji list-tagged --inherit --latest TAG``
-
+  If packages section is declared but is empty, the compose will include all
```

Comment:
Double-quotes around `packages` for consistency with the other added paragraph?

Reply:
Fixed, and declaration changed to key for similar consistency.